tools/releaseBuild/yaml/releaseBuild.yml

```diff
@@ -30,10 +30,9 @@ stages:
 
   - job: SignWin
     pool:
-      name: PowerShell
+      name: Package ES Standard Build
       demands:
       - DotNetFramework
```

Comment:
You can remove this too, it's dynamically added on windows if you use the PowerShell taks
```suggestion
```